src/Avalonia.Animation/ClockBase.cs

```diff
@@ -8,17 +8,14 @@ namespace Avalonia.Animation
 {
     public class ClockBase : IClock
     {
-        private ClockObservable _observable;
-
-        private IObservable<TimeSpan> _connectedObservable;
+        private readonly ClockObservable _observable;
 
         private TimeSpan? _previousTime;
         private TimeSpan _internalTime;
 
         protected ClockBase()
         {
             _observable = new ClockObservable();
-            _connectedObservable = _observable.Publish().RefCount();
```

Comment:
Apart from adding 6 layers of indirection this was not doing anything useful. This observable does not initialize any resources when subscribed. I guess it was remnant of times where subscribing to clock was creating an actual timer.